### PR TITLE
Wiki id and ttile

### PR DESCRIPTION
**Pull request by AshakaE**
Unpaginted rest API for wiki ID and Title 

```bash
curl  -X GET \
  'https://<API_URL>/wiki' 
```

### CODE REVIEW

File: src/App/Wiki/controllers/wiki.controller.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Controller, Get } from '@nestjs/common'
+import WikiService from '../wiki.service'
+
+@Controller('wiki')
+class WikiController {
+  constructor(private wikiService: WikiService) {}
+
+  @Get()
+  async wikiIdAndTitle() {
+    return this.wikiService.getWikiIdAndTitle()
```
Comment (kesar) [inline src/App/Wiki/controllers/wiki.controller.ts:10]: can we add some cache on top if its not super complex ? like refresh each hour or so

Comment (AshakaE) [reply src/App/Wiki/controllers/wiki.controller.ts:10]: Most definitely, it will cache in 4 processes though

**Comment by Royal-lobster** [comment]
what is the API_URL 😅 ?

**Comment by AshakaE** [comment]
> what is the API_URL 😅 ?

the "api url" without /graphql so 'https://localhost:5432/wiki'

**Comment by Royal-lobster** [comment]
Hosted url ? i am checking https://graph.everipedia.org/wiki but its 404

**Pull request merged**